css/style.css

```diff
@@ -108,48 +108,43 @@ ul {
 }
 
 .info {
-    display: block;
+    width: 1020px;
+    margin: auto;
     text-align: center;
     padding-top: 40px;
 }
 
 .info__title {
+    color: #ff3333;
     font-size: 30px;
-    color: rgb(255, 51, 51);
     line-height: 1.2;
-    margin-left: 50%;
-    transform: translateX(-50%);
-    margin-bottom: 30px;
+    margin: auto;
+    margin-bottom: 19px;
 }
 
 .line {
-    display: block;
-    margin-top: -20px;
-    position: absolute;
-    left: 50%;
-    transform: translateX(-50%);
+    background-color: #ff3333;
+    margin: auto;
     width: 95px;
     height: 3px;
-    background-color: rgb(255, 51, 51);
 }
 
 .info__content {
-    display: flex;
-    margin-left: 180px;
     font-size: 20px;
-    color: rgb(0, 0, 0);
     line-height: 1.2;
+    margin: auto;
+    margin-top: 24px;
+    margin-bottom: 56px;
     width: 660px;
-    padding-bottom: 56px;
 }
 
 .table__team {
```

Comment:
Where is main block '.table' for element '.table__team'? Not correct class name. The name is misleading, I expect there will be a table, but in fact there is none.